Guided Investigation - Azure WAF SQLI.ipynb

```diff
@@ -0,0 +1,792 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #19.        sqli_analytics = analytics[(analytics['properties.query'].str.contains("FrontDoorWebApplicationFirewallLog") | analytics['properties.query'].str.contains("ApplicationGatewayFirewallLog")) &amp; (analytics['properties.query'].str.contains("SQLI") | analytics['properties.query'].str.contains("SQL Injection"))]
</pre><p>I thought you needed parentheses around every sub-expression if joined with | or &amp;. </p><p>The first 2 or'd expressions don't have that.</p><p>Might also be easier to read if split on multiiple lines</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Did some extra testing and it appear to work as intended - I've split over 2 lines to improve readability.